k8s.gcr.io/lib.sh

```diff
@@ -34,6 +34,11 @@ function color() {
 # The group that admins all GCR repos.
 GCR_ADMINS="k8s-infra-gcr-admins@googlegroups.com"
 
+# The group that admins all GCS buckets.
+# TODO: Do we want a separate group from GCR??
```

Comment:
We could rename to "k8s-infra-storage-admins@googlegroups.com" - it's less obvious for the admins than it is for the staging, IMO